### PR TITLE
Perrys tree limb

### PR DESCRIPTION
**Pull request by perry9999**
checked for zero start miles, changed starting mpw to stop the divide by zero error.

### CODE REVIEW

File: logic/createGoogleCalendar.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ let respond;
 showLogs = true; //true = shows lots of console log stuff, false shows "ERROR containing messages only"
 function logthis(stuff) {
   if (showLogs || stuff.includes("ERROR")) {
-    console.log(stuff);
+    logthis(stuff);
```
Comment (cmlinac) [inline logic/createGoogleCalendar.js]: this is an infinite loop

File: client/src/components/Forms.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ const StartForm = (props) => {
             <Label htmlFor="mpw">How many Miles Per Week (MPW) do you currently
                   run?</Label>
             <Input  onChange={props.onChange} type="select" name="mpw">
-              <option value="0">0</option>
+              <option value="0">1</option>
```
Comment (cmlinac) [inline client/src/components/Forms.js]: You want to set the value, not the inner text. Value is what is sent when submitted, inner text is what shows up on the site.

File: logic/createGoogleCalendar.js (hunk 1 above)
Comment (perry9999) [reply logic/createGoogleCalendar.js]: What is holding up my merges? This is fixed..

**Comment by perry9999** [comment]
Please check and merge!

**Pull request merged**